src/Workspaces/MSBuildTest/Resources/ProjectFiles/CSharp/DuplicateProjectReference.csproj

```diff
@@ -32,12 +32,10 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System" />
+    <Reference Include="X\..\EmptyLibrary.dll" />
```

Comment:
Does the file need to be renamed? (I can't tell if the meaning is "this has a duplicate <ProjectReference>" or "this is a project with duplicate references")

Reply:
You mean the csproj file? I can rename it to DuplicateProjectAndMetadataReferences.csproj